glean-core/Cargo.toml

```diff
@@ -34,14 +34,14 @@ rkv = { version = "0.19.0", default-features = false }
 bincode = "1.2.1"
 log = "0.4.8"
 uuid = { version = "1.0", features = ["v4"] }
-chrono = { version = "0.4.10", features = ["serde"] }
+chrono = { version = "0.4.41", features = ["serde"] }
 once_cell = "1.18.0"
 flate2 = "1.0.19"
 zeitstempel = "0.1.0"
 crossbeam-channel = "0.5"
-thiserror = "1.0.4"
+thiserror = "2"
```

Comment:
Handled in https://github.com/mozilla/glean/pull/3163
We need other updates first.